src/client/src/orangehrmClaimPlugin/pages/SaveClaimEvent.vue

```diff
@@ -112,15 +112,14 @@ export default {
   },
 
   created() {
-    //TODO: Refctor this code to use validation API
     this.isLoading = true;
     this.http
       .getAll({limit: 0})
       .then((response) => {
         const {data} = response.data;
         this.rules.name.push((v) => {
           const index = data.findIndex(
-            (item) => item.name.toLowerCase() == v.toLowerCase(),
+            (item) => item.name.toLowerCase() == v.trim().toLowerCase(),
```

Comment:
@ManulMax check this. need to validate uniqueness in BE validation

Reply:
We have to validate in BE. @DulsaraNethmin you can check src/plugins/orangehrmAdminPlugin/Api/UserAPI.php
`new Rule(Rules::ENTITY_UNIQUE_PROPERTY, $uniquePropertyParams)`

Reply:
BE validation implemented